sql-statements/sql-statement-kill.md

```diff
@@ -33,6 +33,31 @@ SHOW PROCESSLIST;
 2 rows in set (0.00 sec)
 ```
 
+可以查询 `INFORMATION_SCHEMA.CLUSTER_PROCESSLIST` 的 `INSTANCE` 列，获取被 kill 的会话连接的 TiDB Server 信息。
+
+{{< copyable "sql" >}}
+
+```sql
+SELECT ID, USER, INSTANCE, INFO FROM INFORMATION_SCHEMA.CLUSTER_PROCESSLIST;
+```
+
+```
++---------------------+------+-----------------+-----------------------------------------------------------------------------+
+| ID | USER | INSTANCE | INFO |
++---------------------+------+-----------------+-----------------------------------------------------------------------------+
+| 1 | root | 127.0.0.1:10082 | SELECT ID, USER, INSTANCE, INFO FROM INFORMATION_SCHEMA.CLUSTER_PROCESSLIST |
+| 2 | root | 127.0.0.1:10080 |  |
++---------------------+------+-----------------+-------------------------------------------------------------
+2 rows in set (0.00 sec)
+
+```
```

Comment:
```suggestion
```sql
+---------------------+------+-----------------+-----------------------------------------------------------------------------+
| ID | USER | INSTANCE | INFO |
+---------------------+------+-----------------+-----------------------------------------------------------------------------+
| 1 | root | 127.0.0.1:10082 | SELECT ID, USER, INSTANCE, INFO FROM INFORMATION_SCHEMA.CLUSTER_PROCESSLIST |
| 2 | root | 127.0.0.1:10080 |  |
+---------------------+------+-----------------+-------------------------------------------------------------
2 rows in set (0.00 sec)
```
```